pkg/discovery/dtofactory/pod_entity_dto_builder.go

```diff
@@ -388,6 +398,19 @@ func (builder *podEntityDTOBuilder) getPodCommoditiesBought(
 		commoditiesBought = append(commoditiesBought, accessComm)
 	}
 
+	// Label commodities
+	for key, value := range pod.Spec.NodeSelector {
+		selector := key + "=" + value
+		labelComm, err := sdkbuilder.NewCommodityDTOBuilder(proto.CommodityDTO_LABEL).
+			Key(selector).
+			Create()
+		if err != nil {
+			return nil, err
+		}
+		glog.V(5).Infof("Adding label commodity for Pod %s with key : %s", pod.Name, selector)
+		commoditiesBought = append(commoditiesBought, labelComm)
+	}
```

Comment:
Similar to node, we need to remove the logic to create `VMPM_ACCESS` bought commodity for labels above.

Reply:
Removed.